pages/thc/[processRef]/review.tsx

```diff
@@ -117,46 +105,10 @@ const ReviewPage: NextPage = () => {
 
   // Long term we want an interface that allows the user to select which tenant
   // to use, but for now we stick to the first tenant in the list.
-  if (selectedTenantId !== tenantIds[0]) {
-    setSelectedTenantId(tenantIds[0]);
+  if (selectedTenantId !== presentTenantIds[0]) {
+    setSelectedTenantId(presentTenantIds[0]);
   }
```

Comment:
I think this can disappear entirely now, along with the `useState` hook that created it?